README.md

```diff
@@ -684,6 +684,12 @@ You should consider scoping those selectors with a library's unique HTML class,
 
 For advanced cases, we recommend to implement one custom seeding per each model (see below).
 
+Import:
+
+```js
+import { givenSteps } from 'ember-cli-yadda-opinionated/test-support';
```

Comment:
This is mostly unrelated to the specific changes here, but just stumbled upon the import path here. Given that the addon is *exclusively* meant to be used in tests, i.e. there is zero runtime code, I wonder if it wouldn't be nicer to omit the `test-support/` part here? Like

```
import { givenSteps } from 'ember-cli-yadda-opinionated';
```

Technically this is possible, while still keeping all the imported code in the `test-support.js` bundle. See 
https://github.com/kaliber5/ember-window-mock/blob/master/index.js#L6-L23

Reply:
Extracted into #23.